utils/config.ts

```diff
@@ -30,6 +37,15 @@ export const IConfig = t.intersection([
 
     isProduction: t.boolean
   }),
+  t.interface({
+    NH_PARTITION_FEATURE_FLAG: enumType<NHPartitionFeatureFlag>(
+      NHPartitionFeatureFlag,
+      "NHPartitionFeatureFlag"
+    ),
```

Comment:
In case you accepted the suggestion above:
```suggestion
    NH_PARTITION_FEATURE_FLAG: NHPartitionFeatureFlag,
```

Reply:
Done as `keyof` in [888e53a](https://github.com/pagopa/io-functions-pushnotifications/pull/17/commits/888e53a33e5c65b25e5e322287cd8407a9389747)